main.go

```diff
@@ -295,18 +305,37 @@ func main() {
 
 	// Setup the context that's going to be used in controllers and for the manager.
 	ctx := ctrl.SetupSignalHandler()
+	registerControllers(ctx, mgr)
+	// +kubebuilder:scaffold:builder
 
-	if err = controllers.NewAzureMachineReconciler(
-		mgr.GetClient(),
-		ctrl.Log.WithName("controllers").WithName("AzureMachine"),
+	if err := mgr.AddReadyzCheck("ping", healthz.Ping); err != nil {
```

Comment:
is it normal that readyz check takes the same args as healthz check?

Reply:
I agree. I was just keeping what is already in HEAD.

https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/bd904772ec98f83631c4528260f0c55a2a6efbcf/main.go#L431-L434

Reply:
Oddly, the same in CAPI:
https://github.com/kubernetes-sigs/cluster-api/blob/45a0abb58cf1bcd65c9b35b0f7e189ac2cae6f0a/main.go#L217-L220